app/views/people/_details_pbs.html.haml

```diff
@@ -4,6 +4,14 @@
 -#  https://github.com/hitobito/hitobito_pbs.
 
 = render_attrs(entry, :pbs_number, :salutation, :correspondence_language,
-                      :grade_of_school, :brother_and_sisters)
+                      :grade_of_school)
+
+%dl.dl-horizontal
+  = labeled(t('people.fields_pbs.siblings_in_layer', layer: parent.layer_group)) do
+    - if entry.has_siblings_in_layer(parent.layer_group)
+      = simple_list entry.siblings_in_layer(parent.layer_group) do |sibling_role|
+        -  assoc_link(sibling_role.person)
+    - else
+      = t('global.associations.no_entry')
```

Comment:
@carlobeltrame macht es hier in der view allenfalls Sinn, gleich die Geschwister anzuzeigen statt nur ja/nein?

![image](https://github.com/hitobito/hitobito_pbs/assets/939106/fe1dbf10-878b-4abe-bdff-56f1fcb7bd4a)

Reply:
Diese Frage geben wir doch gerne weiter an @simfeld

Reply:
@diegosteiner ja, das macht aus meiner Sicht durchaus Sinn. Gerne so umsetzen falls ohne grosse Umstände möglich.

Reply:
@simfeld und wie würdest du diese Geschwister dann gerne in den Excel-Exports und in der API dargestellt haben?

Reply:
Müssen wir das? Ich habe die Frage als limitiert auf die view einer einzelnen Person betrachtet. Also das Feld wäre weiterhin boolean, es gibt einfach eine Zusatzinformation an dieser Stelle. Wie gesagt, falls das mehr Fragen aufwirft als es löst, let's not do it.

Reply:
:+1: also um ganz klar zu sein: In Export und API vorerst nur das Boolean Feld. In der Personen-/Participation-Detailansicht die tatsächlichen Geschwister auflisten, aber kein Boolean-Feld? Oder beides?

@diegosteiner die Personen sollten wir dann nur verlinken wenn man auch Leserechte hat, ansonsten nur den Namen als Text anzeigen. Möglicherweise passiert das schon automatisch. Falls nicht gibts irgendwo einen Helper dafür, suche den noch raus...

Reply:
> In der Personen-/Participation-Detailansicht die tatsächlichen Geschwister auflisten, aber kein Boolean-Feld? Oder beides?

Von mir aus gesehen beides, sonst führt es eher zu Verwirrung. Also "ja" und dann die Liste der Geschwister oder ansonsten "nein".

Reply:
Danke für die Inputs @simfeld und @carlobeltrame , dann mache ich das wie folgt:

- API und Export bleiben boolean Feld
- In dieser View boolean und Auflistung
- Verlinkt nur dann wenn Leserechte auf die Person

Reply:
> 👍 also um ganz klar zu sein: In Export und API vorerst nur das Boolean Feld. In der Personen-/Participation-Detailansicht die tatsächlichen Geschwister auflisten, aber kein Boolean-Feld? Oder beides?
> 
> @diegosteiner die Personen sollten wir dann nur verlinken wenn man auch Leserechte hat, ansonsten nur den Namen als Text anzeigen. Möglicherweise passiert das schon automatisch. Falls nicht gibts irgendwo einen Helper dafür, suche den noch raus...

@carlobeltrame scheint bereits so implementiert zu sein: https://github.com/hitobito/hitobito/blob/e08e960330fc0c72603adedf43e7fdce96e60947/app/helpers/format_helper.rb#L220